packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -98,24 +114,70 @@ class CodeSizeBodyState extends State<CodeSizeBody>
               controller: _tabController,
               tabs: tabs,
             ),
+            Row(
+              children: [
+                _buildDiffTreeTypeDropdown(currentTab),
```

Comment:
shouldn't we only show this if the current tab is the diff tab?

Reply:
Done